ctx.go

```diff
@@ -894,6 +901,20 @@ func (c *Ctx) JSON(data interface{}) error {
 	return nil
 }
 
+// CustomJSON converts any interface or string to JSON,
+// similar to the JSON method.
+// This method also sets the content header to the given
+// ctype string.
+func (c *Ctx) CustomJSON(data interface{}, ctype string) error {
```

Comment:
why not do something like this ?

```go
func (c *Ctx) CustomJSON(data interface{}, ctype string) error {
    err := c.JSON(data)
    c.fasthttp.Response.Header.SetContentType(ctype)
    return err
}
```

Reply:
can you also test a signature with variadic params

func (c *Ctx) JSON(data interface{}, ctype ...string) error {
    raw, err := c.app.config.JSONEncoder(data)
	if err != nil {
		return err
	}
	c.fasthttp.Response.SetBodyRaw(raw)
	
      if len(ctype) > 0 {
        	c.fasthttp.Response.Header.SetContentType(ctype[0])
    } else {
         c.fasthttp.Response.Header.SetContentType(MIMEApplicationJSON)
    }
   return nil
}
```
if this does not affect the performance we prefer this

Reply:
@rhburt

Reply:
I've changed all the signatures to accept variadic ctype parameters and have included the relevant benchmark numbers.